students/Pirouz_N/lesson03/mailroom.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+"""
+Purpose: Mailroom Part 1 python certificate from UW
+Author: Pirouz Naghavi
+Date: 07/03/2020
+"""
+
+donor_db = [("William Gates, III", [653772.32, 12.17]),
+            ("Jeff Bezos", [877.33]),
+            ("Paul Allen", [663.23, 43.87, 1.32]),
+            ("Mark Zuckerberg", [1663.23, 4300.87, 10432.0]),
+            ]
+
+
+def main_menu():
+    """"This function makes the menu portion of the program work."""
+    while True:
+        option = input('Please select of the three options: Send a Thank You, Create a Report or Quit.\n'
+                       'Type 1 and hit enter if you wish to Send a Thank You\n'
+                       'Type 2 and hit enter if you wish to Create a Report\n'
+                       'Type 3 and hit enter if you wish to Quit\n'
+                       'Please make your selection:\n')
+        if option == '1':
+            send_thank_you()
+        elif option == '2':
+            create_report()
+        elif option == '3':
+            break
+        else:
+            print('Please only type 1 or 2 or 3 and hit enter. Any other option will not be acceptable.')
+
+
+def create_report():
+    """"This function will create a report."""
+    # Printing table
+    print('{:<30.30s}\t|{:^16.16s}\t|{:^12.12s}\t|{:^16.16s}'
+          .format('Donor Name', 'Total Given', 'Num Gifts', 'Average Gift'))
+    print('------------------------------------------------------------------------------------------------')
+
+    # printing donor table
+    for donor in donor_db:
+        # printing donor table
+        print(
+            '{:<30.30s}\t ${:>15.15s}\t{:>12.12s}\t ${:>15.15s}'
+            .format(donor[0], format(float(sum(donor[1])), '.2f'), str(len(donor[1])),
+                    format(float(sum(donor[1]) / len(donor[1])), '.2f')))
+    print('\n')
```

Comment:
this report should be sorted, please check assignment requirements.

Reply:
Thank you for pointing that out. So sorry I missed that. I have fixed it and will sent you a pull request ASAP.

Best Regards
Pirouz Naghavi